src/base/bittorrent/private/filterparserthread.cpp

```diff
@@ -202,30 +204,38 @@ int FilterParserThread::parseDATFilterFile()
             int endOfIPRange = ((firstComma == -1) ? (endOfLine - 1) : (firstComma - 1));
             int delimIP = findAndNullDelimiter(buffer.data(), '-', start, endOfIPRange);
             if (delimIP == -1) {
-                LogMsg(tr("IP filter line %1 is malformed.").arg(nbLine), Log::CRITICAL);
+                parseErrorCount++;
+                if (parseErrorCount < MAX_LOG_LINES)
+                    LogMsg(tr("IP filter line %1 is malformed.").arg(nbLine), Log::CRITICAL);
```

Comment:
It should be possible to use a lambda here:
```c++
const auto addLog = [&parseErrorCount](const QString &msg)
{
    ++parseErrorCount;
    if (parseErrorCount < MAX_LOG_LINES)
        LogMsg(msg, Log::CRITICAL);
};

// usage:
addLog(tr("IP filter line %1 is malformed.").arg(nbLine));
```

Reply:
What's the point?

Reply:
Ahh... this is repeated many times. Then Yes.